http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/RuntimeTemplateValuesSpec.scala

```diff
@@ -54,6 +60,7 @@ class RuntimeTemplateValuesSpec extends LeonardoTestSuite with AnyFlatSpecLike {
       result.jupyterStartUserScriptOutputUri shouldBe RuntimeTemplateValues
         .jupyterUserStartScriptOutputUriPath(CommonTestData.stagingBucketName, now)
         .toUri
+      result.updateWelder shouldBe "true"
```

Comment:
do we support upgrading crypto mining container out of the box as well? if not, might worth log a ticket

Reply:
Good question.. we don't support it, but may be a good idea. On the other hand it's pretty easy to just manually delete crypto-VMs if there are any and we need to make a change to this tool.